doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -160,11 +163,11 @@ There are pros and cons to using both:
       it's a bug), but 3rd party code based on NumPy may not honor type
       preservation like NumPy does.
    -  ``:)`` ``A*B`` is matrix multiplication, so more convenient for
-      linear algebra.
+      linear algebra (only applies to Python < 3.5).
```

Comment:
This is not clear, and suggests that A*B is not matrix multiplication in python 3.